src/services/decoders/ClpIrDecoder/index.ts

```diff
@@ -79,6 +79,25 @@ class ClpIrDecoder implements Decoder {
         return new ClpIrDecoder(module, dataArray, decoderOptions);
     }
 
+    /**
+     * Formats unstructured log events by prepending formatted timestamp to each message.
+     *
+     * @param results
+     * @return The formatted log events.
+     */
+    static #formatUnstructuredResults = (results: DecodeResult[]): DecodeResult[] => {
```

Comment:
I think according to our [declaration order](https://docs.yscope.com/dev-guide/contrib-guides-general.html#declaration-order) guidelines, this should go after the public methods, right?

Reply:
hey @coderabbitai , could you create an issue at https://github.com/y-scope/eslint-config-yscope to track this linter configuration inconsistency with our organization's styling rules?